contributing/translations.md

```diff
@@ -53,6 +60,23 @@ Translations are managed on [Transifex](https://www.Transifex.com/borgbase/vorta
 - **Archive** (not snapshot) = result of `borg create` execution, an identifier to find a
   collection of files in a repo, as they existed at a past point in time.
 
+Not translated, always in English:
+- log messages (log file as well as log output on console or elsewhere)
+- other console output, print().
+- docs
+- py source code, comments, docstrings
+
+Translated:
+- GUI texts / messages
+
+Add new strings for translation:
+- In Qt (sub)classes, use self.tr("English string"), scope will
+  be the instance class name.
+- Elsewhere use vorta.i18n.translate("scopename", "English string")
+- To only mark for string extraction, but not immediately translate, use 
+  vorta.i18n.trans_late function. Later, to translate, use vorta.i18n.translate (giving same scope).
+
```

Comment:
Add note here about string has to be marked with `trans_late` for `translate` to work later

Reply:
Sidenote: Why do we have separate `trans_late` and `translate`?

Reply:
Not sure why Thomas did it like that. One is a dummy to mark the string. Maybe we can make it just one in the future.

I'll revisit this later and see if this needs changing or merge the functions.

```
def translate(*args, **kwargs):
    """
    small wrapper around QCoreApplication.translate()
    """
    global application  # see init_translation
    return application.translate(*args, **kwargs)


def trans_late(scope, text):
    """
    dummy translate function - only purpose is to mark scope/text for message extraction.

    later, at another place, there will be a translate(scope, text) call, with same
    scope and text, potentially not literal scope, not literal text, but from a variable.
    """
    return text
```

Reply:
Ah, I think he needed this to translate e.g. settings.